lib/Factor.js

```diff
@@ -34,4 +34,18 @@ Factor.getPrimeFactors = function(number){
   return factors;
 };
 
+Factor.getFactorPairs = function(number){
```

Comment:
oh yeah! we'll need this for quadratics

docstring here? also should we handle number being negative here or before calling this?